extensions/adapters/raster/src/main/java/org/locationtech/geowave/adapter/raster/operations/InstallGdalCommand.java

```diff
@@ -19,13 +21,19 @@
 @GeowaveOperation(name = "installgdal", parentOperation = RasterSection.class)
 @Parameters(commandDescription = "Install GDAL by downloading native libraries")
 public class InstallGdalCommand extends DefaultOperation implements Command {
+  private static final String DEFAULT_DOWNLOAD_DIR = "lib/utilities/gdal";
 
   @Parameter(names = "--dir", description = "The download directory", required = false)
-  private String downloadDirectory = "./lib/utilities/gdal";
+  private String downloadDirectory = null;
 
   @Override
   public void execute(final OperationParams params) throws Exception {
-    InstallGdal.main(new String[] {downloadDirectory});
+    if (downloadDirectory == null) {
+      final String homeDirectory = System.getProperty("geowave.home", ".");
+      final Path path = Paths.get(homeDirectory, DEFAULT_DOWNLOAD_DIR);
+      InstallGdal.main(new String[] {path.toString()});
```

Comment:
Move `InstallGdal.main` function invocation out of conditional to avoid code duplication.  The `else` can be removed entirely if the `if` statement just updates the `downloadDirectory` variable.